### PR TITLE
[docs] Add jsonb upsert

### PR DESCRIPTION
**Pull request by ddhodge**
Add jsonb upsert info for Spark connector (Java) #9805

https://deploy-preview-9852--infallible-bardeen-164bc9.netlify.app/latest/integrations/apache-spark/java/

### CODE REVIEW

File: docs/content/latest/integrations/apache-spark/java.md [hunk 1]
```diff
@@ -157,9 +163,94 @@ assertTrue(explain_text.contains(
...
+updatedRows.write().format("org.apache.spark.sql.cassandra").option("keyspace", KEYSPACE)
+    .option("table", CDR_TABLE)
+    .option("spark.cassandra.mergeable.json.column.mapping", "dl,rsrp,sinr,ul:usage")
+```
+
+##### spark.cassandra.output.ignoreNulls
+
+Set to true to cause all null values to be left as unset rather than bound.
 
+For example, for `{"dl": null, "ul":"foo"}`, if `spark.cassandra.output.ignoreNulls` is specified, the value null would be omitted for field `dl` when writing (merging) into the corresponding JSONB column, as though field `dl` didn't appear in the input.
```
Comment (polarweasel) [inline docs/content/latest/integrations/apache-spark/java.md:256]:
```suggestion
For example, for `{"dl": null, "ul":"foo"}`, if `spark.cassandra.output.ignoreNulls` is specified, only the `ul` field gets written (merged) into the corresponding JSONB column, and the `ul` field is ignored.
```

**Review by polarweasel — CHANGES_REQUESTED** [review]

**Comment by polarweasel** [comment]
@nchandrappa please have a look at my comments...

File: docs/content/latest/integrations/apache-spark/java.md [hunk 2]
```diff
@@ -157,9 +163,94 @@ assertTrue(explain_text.contains(
...
+##### spark.cassandra.json.quoteValueString
+ 
+Specifies whether the JSONB field values should be quoted as string. Defaults to false (since this is not space efficient).
+ 
+The option allows you to preserve floating point precision for JSONB fields. For un-quoted non-numeric values, double quotes are added. Otherwise the values would be rejected by YCQL.
+
+Here is an example - note the quotes around 100.1:
+
+```json
+"dl":1122,"rsrp":["abc","rsrq":null,"sinr":"100.1"]
```
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/latest/integrations/apache-spark/java.md:221]: @nchandrappa

[the remark this answers is not in the document]
Comment (nchandrappa) [reply docs/content/latest/integrations/apache-spark/java.md:221]: Yes, let's update the JSON snipped to a well-formed one Alex has provided above. Basically, snippet is trying to show quotes around json values.

**Comment by nchandrappa** [comment]
other changes look good.

**Review by polarweasel — APPROVED** [review]
LGTM

**Pull request merged**